extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -798,68 +801,111 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
   }
 
   private checkReciepientsKeys = async () => {
-    for (const recipientEl of this.addedRecipients.filter(r => r.element.className.includes('no_pgp'))) {
+    for (const recipientEl of this.addedRecipients.filter(
+      r => r.element.className.includes('no_pgp'))) {
       const email = $(recipientEl).text().trim();
-      const [dbContact] = await ContactStore.get(undefined, [email]);
-      if (dbContact) {
-        recipientEl.element.classList.remove('no_pgp');
-        await this.renderPubkeyResult(recipientEl, dbContact);
+      const dbContacts = await ContactStore.get(undefined, [email]);
+      if (dbContacts && dbContacts.length) {
+        const realDbContacts = dbContacts.filter(contact => contact !== undefined);
+        if (realDbContacts && realDbContacts.length) {
+          recipientEl.element.classList.remove('no_pgp');
+          await this.renderPubkeyResult(recipientEl, realDbContacts as Contact[]);
+        }
```

Comment:
This code makes me think we should start calling `Contact` something else. Probably `DbPubkey`, as the meaning of the structure has changed over time. That would be for another issue, of course.

Reply:
Posted #4005